src/app/components/header.tsx

```diff
@@ -11,57 +11,59 @@ import {
 } from '@nextui-org/react';
 import { useState } from 'react';
 
+const menu = [
+  {
+    href: '/rsvp',
+    title: 'RSVP',
+  },
+  {
+    href: '/schedule',
+    title: 'Schedule',
+  },
+  {
+    href: '/travel',
+    title: 'Travel & Stay'
+  },
+  {
+    href: '/registry',
+    title: 'Registry'
+  }
+];
+
 export function Header() {
   const [isMenuOpen, setIsMenuOpen] = useState(false);
 
   return (
     <Navbar onMenuOpenChange={setIsMenuOpen}>
+      {/* Toggle Menu for  mobile devices */}
       <NavbarContent>
         <NavbarMenuToggle
           aria-label={isMenuOpen ? 'Close menu' : 'Open menu'}
           className='sm:hidden'
         />
         <NavbarBrand>
-          <Link href='/'>💍 J+O</Link>
+          <Link href='/' color="foreground"><p className="font-bold text-inherit">💍 J+O</p></Link>
         </NavbarBrand>
       </NavbarContent>
-
+      {/* Menu Content for large devices */}
       <NavbarContent className='hidden sm:flex gap-4' justify='end'>
-        <NavbarItem>
-          <Link href='/rsvp'>RSVP</Link>
-        </NavbarItem>
-        <NavbarItem>
-          <Link href='/schedule'>Schedule</Link>
-        </NavbarItem>
-        <NavbarItem>
-          <Link href='/travel'>Travel & Stay</Link>
-        </NavbarItem>
-        <NavbarItem>
-          <Link href='/registry'>Registry</Link>
-        </NavbarItem>
+        {
+          menu.map((item, idx) => (
+            <NavbarItem key={`${item.title}-${idx}`}>
+              <Link href={item.href} size="lg" color="foreground">{item.title}</Link>
+            </NavbarItem>
+          ))
+        }
       </NavbarContent>
-
+      {/* Menu Content for small devices */}
```

Comment:
I'd prefer to have these broken up into semantically named components rather than having comments, but it's nbd either way to me, cutie. <3

Reply:
They're not really separate components, they can't be re-used in any way and only apply in the context of this navbar. The way it's designed in NextUI these exist next to each other and are just hidden or displayed with css classes. Does that change anything?